about.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>About - Alex Brown</title>
+    <link rel=StyleSheet href="stylesheets/normalize.css" type="text/css" media=screen>
+    <link rel=StyleSheet href="stylesheets/about-style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Rock+Salt" rel="stylesheet">
+  </head>
+
+  <body class="background">
+
+    <nav class="nav-container">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="portfolio.html">Portfolio</a>
+        <a href="blog.html">Blog</a>
+    </nav>
+
+    <section class="main-container">
+      <div class="items">
+        <h1>About</h1>
+        <p>
+          Pommy ipsum picalilly good old fashioned knees up the black death bobby on his bill, sod's law macca porky-pies nutter know your onions. Jolly hockey sticks scones nutter penny-dreadful tad have a bash apple and pears, scouser one feels that upper class anorak.
+        </p>
+      </div>
+
+      <footer>
+        <h2 class="sub-title" id="contact-header">Contact:</h2>
+        <div class="contact-info">
```

Comment:
Another small suggestion, icons would do better for github, linkedin and email rather than text.